nemo/collections/asr/models/hybrid_asr_tts_models.py

```diff
@@ -100,20 +101,25 @@ class ASRModelTypes(PrettyStrEnum):
 
         RNNT_BPE = "rnnt_bpe"
         CTC_BPE = "ctc_bpe"
+        HYBRID_RNNT_CTC_BPE = "hybrid_rnnt_ctc_bpe"
 
         @classmethod
         def from_asr_model(cls, model: Any):
             if isinstance(model, EncDecRNNTBPEModel):
                 return cls.RNNT_BPE
             if isinstance(model, EncDecCTCModelBPE):
                 return cls.CTC_BPE
+            if isinstance(model, EncDecHybridRNNTCTCBPEModel):
+                return cls.HYBRID_RNNT_CTC_BPE
             raise ValueError(f"Unsupported model type: {type(model)}")
 
         def get_asr_cls(self):
             if self == self.RNNT_BPE:
                 return EncDecRNNTBPEModel
             if self == self.CTC_BPE:
                 return EncDecCTCModelBPE
+            if self == self.HYBRID_RNNT_CTC_BPE:
```

Comment:
What is this function used for ? You can get the class of the ASR model as follows - model_utils.import_from_classpath(asr_model.cfg.target)

Reply:
This function is needed when we instantiate hybrid ASR-TTS model from ASR config. There is no `target` in configs: https://github.com/NVIDIA/NeMo/blob/main/examples/asr/conf/conformer/conformer_transducer_bpe.yaml
So, I need to pass something to instantiate the model. 

In training scripts:
```shell
python speech_to_text_bpe_with_text.py \
...
++asr_model_type=<rnnt_bpe, ctc_bpe or hybrid_rnnt_ctc_bpe> \
...
```

Reply:
Ah understood